models/omen/gnosis/omen_gnosis_trades.sql

```diff
@@ -0,0 +1,123 @@
+{{ config(
+    schema = 'omen_gnosis',
+    alias = 'trades',
+    
+    partition_by = ['block_day'],
```

Comment:
i would recommend removing the partition column altogether. we don't want to over-partition, or it'll make performance worse. you can see how small each day is:
```
select
  block_day,
  count(1)
from
  test_schema.git_dunesql_0191ef6_omen_gnosis_trades
group by
  block_day
```

Reply:
this likely applies to both models